pkgs/development/python-modules/cusfpredict/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  deprecated,
+  hopcroftkarp,
+  joblib,
+  matplotlib,
+  numpy,
+  scikit-learn,
+  scipy,
+  pytestCheckHook,
+  pythonAtLeast,
+  pythonOlder,
+}:
+buildPythonPackage rec {
+  pname = "cusfpredict";
+  version = "0.2.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "darksidelemm";
+    repo = "cusf_predictor_wrapper";
+    rev = "f4352834a037e3e2bf01a3fd7d5a25aa482e27c6";
+    hash = "sha256-C8/5x8tim6s0hWgCC7LpN1hesdVME5kpQFqDTEyXHtg=";
+  };
+
+  pythonImportsCheck = [ "cusfpredict" ];
+
```

Comment:
Can you please mention explicitly why there are no tests?

Reply:
I'm not a python developer. What do you want me to add here?

Reply:
I'm not a python developer either... but basically, each Python derivation explicitly mention a note when no tests are triggered (`doCheck = false;`).
Can you please check if the Python project you're trying to add in here contains tests? And if yes, please have a look at other Python derivations to get some inspirations.

Reply:
Looks like there are no tests in this package so I will add `doCheck = false;`

Reply:
Ok, make sure to add a note, something like: `# No tests available`